frontend/pages/index.tsx

```diff
@@ -0,0 +1,24 @@
+import Head from 'next/head'
+import { Inter } from 'next/font/google'
+import styles from '@/styles/Home.module.css'
+import PartsView from '../src/components/PartsView';
+import Selector from '../src/components/Selector';
+
+const inter = Inter({ subsets: ['latin'] })
+
+export default function Home() {
+  return (
+    <>
+      <Head>
+        <title>PC Parts Selector</title>
+        <meta name="description" content="Choose the best part for your PC!" />
```

Comment:
@exdorf @IntiveMichal I think this is fine though, because this is not HTML - this is JSX  and JSX allows closing tags (in fact, you will be in trouble if you don't close them).
I checked inside the rendered page and the above code renders as

```
<head>
<meta charset="utf-8">
<title>PC Parts Selector</title>
<meta name="description" content="Choose the best part for your PC!">
<meta name="viewport" content="width=device-width, initial-scale=1"><link rel="icon" href="/favicon.ico">
<meta name="next-head-count" content="5">
</head>
```

so this JSX does compile to proper HTML!

Reply:
Ok, great!